theano/gpuarray/opt.py

```diff
@@ -2030,6 +2030,11 @@ def local_gpu_cholesky(op, context_name, inputs, outputs):
 matrix_ops_db = LocalGroupDB()
 matrix_ops_db2 = LocalGroupDB(local_opt=theano.gof.opt.GraphToGPULocalOptGroup)
 matrix_ops_db2.__name__ = "matrix_ops_db2"
+
+# For Cholesky decomposition, magma is slower than cusolver (tested for
+# matrices of size 1000). Thus, cusolver is prioritized during graph
+# optimizations. To explicitly use magma, you should disable cusolver using
+# `optimizer_excluding=cusolver` in Theano config.
```

Comment:
Can you add which version of magma and cusolve you compared? benchmark are heavily depending on the version, so it will help if in the futur this changes.